webpack.config.js

```diff
@@ -4,11 +4,21 @@ const config = {
 	...defaultConfig,
 
 	entry: {
-		'quick-editor': './src/quick-editor/index.ts',
-		block: './src/block/index.js',
+		'quick-editor': './src/quick-editor',
+		block: './src/block',
 		'block-view': './src/block/view.js',
-		hovercards: './src/hovercards/index.ts',
-		discussion: './src/discussion/index.ts',
+		'new-block': './src/new-block',
+		'new-block-view': './src/new-block/view.ts',
+		'block-column': './src/new-block/blocks/column',
+		'block-image': './src/new-block/blocks/image',
+		'block-name': './src/new-block/blocks/name',
+		'block-paragraph': './src/new-block/blocks/paragraph',
+		'block-link': './src/new-block/blocks/link',
```

Comment:
You have added the SCSS files directly to webpack here, which seems unusual. Can we not build the SCSS as part of the standard import process?

Reply:
I am planning the following architecture to load and manage styles for both the editor and the frontend, especially as we add more classes for patterns:

- [For both](https://github.com/Automattic/gravatar-enhanced/pull/9/files#diff-25a7bcd80a4be2bf8e47621c3b67b35a95d3daa7d9f2399dfe3be5e99323288bR49): [shared.scss](https://github.com/Automattic/gravatar-enhanced/pull/9/files#diff-263c65cd8a931412e80ea6f59751e6b66ccd473d2d046f91300659a35d77423d)
- [For the editor](https://github.com/Automattic/gravatar-enhanced/pull/9/files#diff-25a7bcd80a4be2bf8e47621c3b67b35a95d3daa7d9f2399dfe3be5e99323288bR51): [edit.scss](https://github.com/Automattic/gravatar-enhanced/pull/9/files#diff-9c472652bbc364d386ba329963409de48660d19656425686f65a3fca33cca3a1)
- [For the frontend](https://github.com/Automattic/gravatar-enhanced/pull/9/files#diff-25a7bcd80a4be2bf8e47621c3b67b35a95d3daa7d9f2399dfe3be5e99323288bR53): [view.scss](https://github.com/Automattic/gravatar-enhanced/pull/9/files#diff-2487359ff9cda997170e0efe998343be959cf06af94fde1270ed479ca05c496f)

If we use the import process, the editor-related CSS files will be combined into one. Therefore, I have added these files as separate Webpack entries so they can be built individually.

Reply:
It's been a long time since I last looked at blocks, but this seems to be different from what I remember. Why do we need a shared file? I think the WP build system should already produce separate files?

Reply:
I think I have found a way to use the import process 😆.

> Why do we need a shared file?

In theory, both the editor and frontend should use the same CSS file. However, in practice, inner blocks in the editor will have [extra `<div>` wrappers and classes](https://github.com/Automattic/gravatar-enhanced/pull/9/files#diff-9c472652bbc364d386ba329963409de48660d19656425686f65a3fca33cca3a1R4). The `shared.scss` file is where common styles for both sides can be placed.

Reply:
Sure, I think I was meaning why there was a shared.css file instead of it being bundled in the other files, but it's gone now anyway!